### PR TITLE
Add bulgarian translation

### PR DESCRIPTION
**Pull request by j4ck-code**

### CODE REVIEW

**Comment by j4ck-code** [comment]
@tulir Please merge or at least check this out?

File: bg/index.html
Comment (tulir) [inline bg/index.html:2]:
```suggestion
<html lang="bg">
```

**Review by tulir — COMMENTED** [review]
You can also add yourself to the page footer and/or `README.md` for credits (see other languages for examples: [es](https://github.com/maunium/dontasktoask.com/blob/master/es/index.html#L88-L89)) and the [`CODEOWNERS` file](https://github.com/maunium/dontasktoask.com/blob/master/CODEOWNERS) if you want to be notified about other people suggesting changes to the translation

(both of those are optional, so if you don't want to do either I can just merge this as-is after the html lang tag is corrected)

**Review by tulir — APPROVED** [review]

**Pull request merged**
